demos/sslecho/A-SSL-Docs.txt

```diff
@@ -0,0 +1,19 @@
+10-Dec-2021		RCW
+
+OpenSSL Wiki: https://wiki.openssl.org/index.php/Main_Page
+
+OpenSSL API Documentation: https://www.openssl.org/docs/man1.1.1/man3/
```

Comment:
I'd suggest pointing to the the top level docs directory rather than a version specific man page, i.e. https://www.openssl.org/docs